Automattic/jetpack
# Jetpack: Add WAF settings notice and Protect CTA

## Description
Adds a info notice to the WAF settings that include Protect CTA

### Other information:

## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1307

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch and start Jurassic Tube
* Install/activate Jetpack
* Proceed to the WAF settings and verify that the notice displays
* Verify that the toggles are still accessible and functional
* Ensure that the Get Protect Buttons CTA redirect to the `#/add-protect` My Jetpack route
* Install Protect and verify that the Jetpack WAF settings now display a message that the Firewall setting are only available in Protect

##Screenshots:

![Screen Shot on 2024-08-06 at 09-56-13](https://github.com/user-attachments/assets/aba6dee7-2ee8-4aab-93eb-9d3d018e0d37)
